src/hammer-vlsi/defaults.yml

```
@@ -83,6 +83,35 @@ vlsi.core:
   # Maximum threads to use in a CAD tool invocation.
   max_threads: 1
 
+#----------------------------------------------------------------------------
+# JOB SUBMISSION
+#----------------------------------------------------------------------------
+vlsi.submit:
+  # - The submit command to use. "none", "local", or null will run on the 
+  #   current host. See hammer_submit_command.py for other options.
+  command: "local"
+  # type: List[Dict[str, Dict[str, Any]]]
+  # The list substitutes settings in order of appearance, and the first 
+  # Dict key is the command. The second dict key is the name of that 
+  # command's setting, followed by whatever type it takes.
+  settings: []
+
+  # how many output lines to buffer in memory. "" for unlimited
+  max_output_lines: null
+  # how many error lines to buffer in memory. "" for unlimited
+  max_error_lines: null
+  # if exit code != 0, or len(error_lines) > 0, raise exception immediately
+  # use "true" or "false"
+  abort_on_error: false
+
+  # take a guess that any time these show up, its an error. individual tools
+  # can override this assumption. use any string that is valid in python regex
+  error_rgxs: ["ERROR:", "error:", "Error:"]
```

Would suggest full names here for readability
```suggestion
  error_regxes: ["ERROR:", "error:", "Error:"]
```